doc/man7/bio.pod

```diff
@@ -47,7 +47,7 @@ in a memory leak.
 Calling BIO_free_all() on a single BIO has the same effect as calling
 BIO_free() on it other than the discarded return value.
 
-Normally the B<type> argument is supplied by a function which returns a
+Normally the I<type> argument is supplied by a function which returns a
 pointer to a BIO_METHOD. There is a naming convention for such functions:
 a source/sink BIO is normally called BIO_s_*() and a filter BIO
```

Comment:
Way up above, `B<ossl_sa_TYPE_>` was used for a partial function name.  Why not here too?

Reply:
We should probably switch to what's on this line.

Reply:
I take that back.  Apparently, the `*` disrupts the usual function detection mechanism...  so we need to help it along.  I'll make this bold.

Reply:
Extra commit added.  Have a look at it, it has a proposal on how to handle function names with a variable part.

Reply:
That should be backported to other "generic" manpages, or this one made to conform.
```
; git grep 'comment generic'
```